src/SDKs/CognitiveServices/dataPlane/Language/LUIS/Programmatic/Microsoft.Azure.CognitiveServices.LUIS.Programmatic.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
```

Comment:
@pcostantini include the comment around the imports of the project file similar to below
https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/CognitiveServices/dataPlane/Search/BingCustomSearch/BingCustomSearch/Microsoft.Azure.CognitiveServices.Search.CustomSearch.csproj#L2

Also include imports that are missing at the bottom of the project file.